src/cassandra/log.rs

```diff
@@ -84,17 +91,64 @@ unsafe extern "C" fn logger_callback(log: *const CassLogMessage, data: *mut raw:
     };
 }
 
-/// Set or unset a logger to receive all Cassandra driver logs.
-pub fn set_logger(logger: Option<slog::Logger>) {
```

Comment:
Removing this is a breaking change, but it doesn't need to be - you can keep this, guarded by cfg(slog).

Reply:
That would still be technically a breaking change, if a user does `cassandra-cpp = { version = "1.1", default-features = false }`, then set_logger would disappear.

I am happy to take either the approach of:
* make the change as you suggest, despite it being breaking
* keep slog as a required dependency and only log is optional

Which do you want?

Reply:
Please go for number 2. Given there are currently (before your change) no default features I hope it's unlikely folks will have turned off default features. I have code that does use slog and doesn't turn off default features, so it would be broken by 1 but fixed by 2. And you surely don't want 3 since you wanted to remove slog from your project (it would be dead weight for anyone just using log).

Thanks!

Reply:
Thanks!